src/main/java/fr/foodlimit/api/Application.java

```diff
@@ -11,6 +11,8 @@
 @SpringBootApplication
 public class Application {
 
+  private String test;
+
   public final static Logger logger = LoggerFactory.getLogger(Application.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)